blog/2024-01-24-nmfs-mentors/index.qmd

```diff
@@ -0,0 +1,99 @@
+
+In fall 2023, we grew as a community and focused on developing teaching, facilitation, and mentoring skills together. We learned coaching skills, which provided a new way to give and receive feedback. We participated in sessions dedicated to active listening, experiential learning, and designing for learners (protip: we are all learners)!
+
+::: {.blockquote-blue}
+> *"The skills we learned as Mentors can be applied to ongoing work and in the workplace without the need to do a training-specific project: in my experience, this aspect is unique to Openscapes. For example, I really valued the fresh perspective of coaching by asking open-ended constructive questions before (or instead of) giving feedback."* **---[Phoebe Woodworth-Jefcoats](https://www.fisheries.noaa.gov/contact/phoebe-woodworth-jefcoats), Research Oceanographer at the Pacific Islands Fisheries Science Center; Honolulu, HI**
+:::
+
+The importance and impact of establishing a Mentors community was evident in topic-based "[Seaside Chats](https://openscapes.org/blog/2019-03-10-seaside-chats/)". Seaside Chats are supportive and psychologically safe spaces. All attendees co-facilitate the meeting and foster open discussions around problems and barriers. Everyone is encouraged to actively contribute to shared notes and report-out documents, rather than leaving facilitation and note-taking responsibilities to one person. During our fall mentor training period, seaside chats worked as follows: one mentor would propose a topic that was important to them and add a seaside chat time to our shared calendar. Mentors from other centers or offices, who had never met before, would join the scheduled session. Despite the distance and barriers, participants easily recognized common challenges, asked questions, shared support solutions, and were able to generalize lessons learned for themselves and others. Between talking, adding to the shared notes, or offering +1's (meaning "I agree" or "me, too") in the meeting's chat, participants learned and practiced several different strategies for connecting with others. During Seaside Chats mentors connected around topics like data visualization, data storage, documentation, and hybrid work environments. But they also connected around difficult topics for which there are no easy solutions, like overcoming frustrating administrative or technical barriers or situations where staff in different regions or offices face very different access to tools essential for reproducible science and data. 
```

Comment:
```suggestion
**The importance and impact of establishing a Mentors community was evident in topic-based "[Seaside Chats](https://openscapes.org/blog/2019-03-10-seaside-chats/)"**. Seaside Chats are supportive and psychologically safe spaces. All attendees co-facilitate the meeting and foster open discussions around problems and barriers. Everyone is encouraged to actively contribute to shared notes and report-out documents, rather than leaving facilitation and note-taking responsibilities to one person. During our fall mentor training period, seaside chats worked as follows: one mentor would propose a topic that was important to them and add a seaside chat time to our shared calendar. Mentors from other centers or offices, who had never met before, would join the scheduled session. Despite the distance and barriers, participants easily recognized common challenges, asked questions, shared support solutions, and were able to generalize lessons learned for themselves and others. Between talking, adding to the shared notes, or offering +1's (meaning "I agree" or "me, too") in the meeting's chat, participants learned and practiced several different strategies for connecting with others. During Seaside Chats mentors connected around topics like data visualization, data storage, documentation, and hybrid work environments. But they also connected around difficult topics for which there are no easy solutions, like overcoming frustrating administrative or technical barriers or situations where staff in different regions or offices face very different access to tools essential for reproducible science and data. 
```